bundle/config/root.go

```diff
@@ -11,8 +11,31 @@ import (
 	"github.com/imdario/mergo"
 )
 
-// FileName is the name of bundle configuration file.
-const FileName = "bundle.yml"
+type ConfigFileNames [4]string
+
+// FileNames is the name of bundle configuration file.
+var FileNames = ConfigFileNames{"databricks.yaml", "databricks.yml", "bundle.yaml", "bundle.yml"}
```

Comment:
Any specific reason for preferring `yaml` over `yml`?

If so can be useful to include as a comment.